### ISSUE TITLE
onClick for commit messages

### ISSUE THREAD

**Issue by Rytiss** [nicoespeon/gitgraph.js#169]
It is currently possible to specify an onClick handler for a commit, however, it only works when clicking on the commit dot. I also need the ability to click on the entire associated message (e.g. hash, message, author) - any way to do that?

**Comment by nicoespeon** [comment]
Hi @Rytiss,

To do that, we need to implement the event when there is a click on the commit message.
It doesn't work yet, but that's a feature to be considered. 

Thanks for raising this 👍

**Comment by nicoespeon** [comment]
@fabien0102 hey, we just talked about it today! Let's do that on v2 👍

**Comment by nicoespeon** [comment]
So, this is technically _not_ resolved for the `gitgraph.js` package… But we released `@gitgraph/react` in v1.0.0-0 and this was implemented here. See [the relevant stories](https://github.com/nicoespeon/gitgraph.js/blob/master/packages/gitgraph-react/src/stories/3-events.stories.tsx).

Thus, I'm closing this issue as we won't maintain the `gitgraph.js` package. There will be a `@gitgraph/js` package soon to replace it. 

We still accept PRs if someone really needs it, but I'd suggest moving to the aforementioned packages. Thanks 🙏
